src/System.Net.Http.WinHttpHandler/tests/FunctionalTests/FakeDiagnosticListenerObserver.cs

```diff
@@ -0,0 +1,71 @@
+﻿using System.Collections.Generic;
```

Comment:
Why do we need this type in source multiple times? In general for these tests, I'd expect it'd be sufficient to just have them for System.Net.Http, as that'll end up covering both the WinHttpHandler and CurlHandler when running those functional tests on each of the platforms.  If we do for some reason need to have separate duplicate tests for WinHttpHandler, we should share the source rather than duplicating it.